packages/moonstone/PictureInGraphics/PictureInGraphics.less

```diff
@@ -0,0 +1,72 @@
+@import '../styles/skin.less';
+@import '../styles/text.less';
+@import '../styles/mixins.less';
+
+@moon-pig-width:      321px;
+@moon-pig-border-width: 6px;
+
+.pictureInGraphics {
+	display: block;
+	position: relative;
+	padding: @moon-pig-border-width;
+	width: @moon-pig-width;
+	height: auto;
+
+	.video {
+		display: block;
+		width: 100%;
+		height: 100%;
+		margin: 0;
+	}
+
+	.image {
+		box-sizing: border-box;
+		background-origin: content-box;
+		display: block;
+		position: absolute;
+		top: 0;
+		left: 0;
+		width: 100%;
+		height: 100%;
+		padding: @moon-pig-border-width;
+		margin: 0;
+	}
+
+	.textOverlay {
+		display: flex;
+		align-items: center;
+		justify-content: center;
+		position: absolute;
+		top: 0;
+		left: 0;
+		height: 100%;
+		width: 100%;
+		padding: @moon-pig-border-width * 3;
+		box-sizing: border-box;
+		.moon-font('MuseoSans', 'Moonstone LG Display Light');
+		font-weight: 900;
+		font-size: 24px;
+		line-height: normal;
+	}
+
+	&:not(:global(.spottable)) {
+		.image {
+			padding: 0;
+		}
+		.textOverlay {
+			padding: @moon-pig-border-width * 2;
+		}
+	}
+
+	:global(.enact-locale-right-to-left) & {
+		.textOverlay,
+		.image {
+			right: 0;
+			left: auto;
+		}
+	}
+}
+
+:not(:global(.spottable))&.pictureInGraphics {
+	padding: 0;
```

Comment:
Since we allow `.pictureInGraphics`, do you think this still should remain as component's requirement?